fic_modules/markdown_modules.py

```diff
@@ -147,15 +148,18 @@ def generate_main_md_table(path_to_files, days_to_generate=1):
     if path_to_files == "git_files":
         extract_json_from_git(json_files, path_to_files, days_to_generate)
         if LOGGER:
-            print("GIT" + successfully_generated)
+            LOGGER.info("GIT" + successfully_generated)
+            # print("GIT" + successfully_generated)
```

Comment:
If we don't require this anymore, it should be removed.

Reply:
Fixed.